### PR TITLE
Add WPF Login form

### PR DESCRIPTION
**Pull request by mihaibuzgau**
Still WIP, do not merge

### CODE REVIEW

**Comment by viovanov** [comment]
verify this please

File: src/CloudFoundry.VisualStudio/Forms/LoginForm.xaml.cs [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            this.IsEnabled = true;
+
+        }
+
+        private void btnCancel_Click(object sender, RoutedEventArgs e)
+        {
+            this.DialogResult = false;
+        }
+
+        private List<string> FormatExceptionMessage(Exception ex)
```
Comment (viovanov) [inline src/CloudFoundry.VisualStudio/Forms/LoginForm.xaml.cs]: This is duplicate code, there's a class called [ErrorFormatter](https://github.com/cloudfoundry-incubator/cf-vs-extension/blob/develop/src/CloudFoundry.VisualStudio/ErrorFormater.cs)

File: src/CloudFoundry.VisualStudio/Forms/LoginForm.xaml.cs [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+            if (string.IsNullOrWhiteSpace(this.tbUrl.Text))
+            {
+                return;
+            }
+
+            if (!this.tbUrl.Text.StartsWith("http://", StringComparison.InvariantCultureIgnoreCase) &&
+                !this.tbUrl.Text.StartsWith("https://", StringComparison.InvariantCultureIgnoreCase))
+            {
+                this.tbUrl.Text = string.Format(CultureInfo.InvariantCulture, "http://{0}", this.tbUrl.Text);
```
Comment (viovanov) [inline src/CloudFoundry.VisualStudio/Forms/LoginForm.xaml.cs]: Should we default to https?

**Comment by carabasdaniel** [comment]
Looks good to me

**Comment by carabasdaniel** [comment]
verify this please

**Pull request merged**